<p>This rule applies whenever an <code>IF</code> statement is followed by one or </p>
<p>more <code>ELSEIF</code> statements, the final <code>ELSEIF</code> shall be followed by an <code>ELSE</code> statement.</p>
<p>The requirement for a final <code>ELSE</code> statement is defensive programming. </p>
<h2>Noncompliant Code Example</h2>
<pre>
	IF FALSE THEN
		CALL doSomething();
	ELSEIF FALSE THEN
		CALL doSomething();
	END IF;
</pre>
<h2>Compliant Solution</h2>
<pre>
	IF FALSE THEN
		CALL doSomething();
	ELSEIF FALSE THEN
		CALL doSomething();
	ELSE
		CALL doSomething();
	END IF;
</pre>