<p>This rule applies to check the parameter count of function</p>
<p>The parameter count of the function should not be higher than the given threshold value.</p>

<h2>Noncompliant Code Example</h2>
<pre>
	CREATE FUNCTION TempFunction(IN Environment REFERENCE,IN EventCode CHARACTER,IN ReplaceValue CHARACTER,
		IN RecipientTo CHARACTER,IN Retries INTEGER,IN SenderFrom CHARACTER,IN ServerHost CHARACTER,IN port INTEGER,
		OUT EventBody CHARACTER, OUT Votes CHARACTER, OUT OutputResult CHARACTER) RETURNS BOOLEAN
	BEGIN
</pre>
<h2>Compliant Solution</h2>
<pre>
	CREATE FUNCTION TempFunction(IN Environment REFERENCE,IN EventCode CHARACTER,IN ReplaceValue CHARACTER,
		OUT OutputResult CHARACTER) RETURNS BOOLEAN
	BEGIN
</pre>