<p>This rule applies to check the parameter count of procedure</p>
<p>The parameter count of the procedure should not be higher than the given threshold value.</p>

<h2>Noncompliant Code Example</h2>
<pre>
	CREATE PROCEDURE ProcedureWithTooManyParams(IN Environment REFERENCE,IN EventCode CHARACTER,IN ReplaceValue CHARACTER,
		IN RecipientTo CHARACTER,IN Retries INTEGER,IN SenderFrom CHARACTER,IN ServerHost CHARACTER,IN port INTEGER,
		OUT EventBody CHARACTER, OUT Votes CHARACTER, OUT OutputResult CHARACTER)
	BEGIN
</pre>
<h2>Compliant Solution</h2>
<pre>
	CREATE PROCEDURE Procedure1(IN Environment REFERENCE,IN EventCode CHARACTER,IN ReplaceValue CHARACTER,
		OUT OutputResult CHARACTER)
	BEGIN
</pre>